services/models.py

```diff
@@ -10,6 +10,9 @@ class Service(models.Model):
     service_type = models.CharField(max_length=32, choices=SERVICE_TYPES, blank=False)
     created_at = models.DateTimeField(auto_now_add=True)
 
+    class Meta:
+        unique_together = ["profile", "service_type"]
```

Comment:
we usually use tuples for most listings in places like this. I think the reason is that it gives you that 0.0001 % increase in security against some weird runtime bugs 😅

Reply:
Kinda makes sense, although official docs prefer list. Let's go with tuples.